pkg/antctl/antctl.go

```diff
@@ -133,9 +136,10 @@ var CommandList = &commandList{
 			transformedResponse: reflect.TypeOf(addressgroup.Response{}),
 		},
 		{
-			use:   "controllerinfo",
-			short: "Print Antrea controller's basic information",
-			long:  "Print Antrea controller's basic information including version, deployment, NetworkPolicy controller, ControllerConditions, etc.",
+			use:     "controllerinfo",
+			aliases: []string{"controllerinfos", "ci"},
+			short:   "Print Antrea controller's basic information",
+			long:    "Print Antrea controller's basic information including version, deployment, NetworkPolicy controller, ControllerConditions, etc.",
```

Comment:
nit: dont see other long messages end with a period "."

Reply:
I guess this period is with etc?